sormas-app/app/src/main/java/de/symeda/sormas/app/backend/common/DatabaseHelper.java

```diff
@@ -2873,6 +2870,149 @@ private void formatRawResultDate(Object[] result, int index) {
 		}
 	}
 
+	private void migrateVaccinationInfo() throws SQLException {
+		// Retrieve all new and unsynchronized cases with vaccinationStatus == VACCINATED from the database
```

Comment:
why new AND unsynchronized cases? From my understanding we can only handle new cases here.
Unsynchronized cases that already exist on the server side will have immunizations generated there and should not be included here.

We should also include the information in the release notes, that vaccination information that are added to an existing case in the mobile app will get lost in case they are not synced at the time when the server is updated. Handling this use case would take too much effort.

PS: The query seems to be correct, but I think there is no need to check for the snapshot.

Reply:
I think that's a misunderstanding, "new" and "unsynchronized" go along here; I've removed the "and" to make it more clear.